### PR TITLE
Pull requests and bug status

### PR DESCRIPTION
**Pull request by markmacmahon**
Providing access to issue creation date and some tweaks for rendering of github pull requests

### CODE REVIEW

File: dash-core/src/main/java/de/axelspringer/ideas/tools/dash/business/jira/rest/Fields.java [hunk 1]
```diff
@@ -19,9 +30,32 @@
...
+
     public Fields() {
     }
 
+    public String getCreated() {
+        return created;
+    }
+
+    public LocalDateTime getCreatedAtDateTime() {
+        //Note this is not precise but mostly good enough - captures the essential fields up to hh:mm
```
Comment (gaffa) [inline dash-core/src/main/java/de/axelspringer/ideas/tools/dash/business/jira/rest/Fields.java]: somehow i dont like this. but i wouldnt mind you leaving it in anyway.

File: dash-core/src/main/java/de/axelspringer/ideas/tools/dash/business/jira/rest/Fields.java [hunk 2]
```diff
@@ -80,4 +114,6 @@ public int hashCode() {
     public String toString() {
         return ToStringBuilder.reflectionToString(this);
     }
+
+
```
Comment (gaffa) [inline dash-core/src/main/java/de/axelspringer/ideas/tools/dash/business/jira/rest/Fields.java]: ++

**Comment by gaffa** [comment]
Merge if you want to.

**Comment by CanardSauvage** [comment]
:+1:

**Pull request merged**